packages/web-config-server/src/apiV1/dataBuilders/generic/count/countCalculatedValuesPerOrgUnit.js

```diff
@@ -0,0 +1,52 @@
+import groupBy from 'lodash.groupby';
+import { checkValueSatisfiesCondition } from '@tupaia/utils';
+import { DataBuilder } from '/apiV1/dataBuilders/DataBuilder';
+import {
+  calculateOperationForAnalytics,
+  getDataElementsFromCalculateOperationConfig,
+} from '/apiV1/dataBuilders/helpers';
+
+class CountCalculatedValuesPerOrgUnit extends DataBuilder {
+  async build() {
+    const { operation, dataClasses } = this.config;
+    const dataElementCodes = getDataElementsFromCalculateOperationConfig(operation);
+    const { results } = await this.fetchAnalytics(dataElementCodes);
+    const analyticsByOrgUnit = groupBy(results, 'organisationUnit');
+    const calculatedValues = Object.values(analyticsByOrgUnit).map(analytics =>
+      calculateOperationForAnalytics(analytics, operation),
+    );
+
+    const data = Object.entries(dataClasses).map(([dataClass, condition]) => {
+      let count = 0;
+      calculatedValues.forEach(value => {
```

Comment:
Could be a tiny bit shorter with `.filter` `.length`, but up to you if you think it is more or less readable.